setup.py

```diff
@@ -277,7 +279,7 @@ def configuration(parent_package='', top_path=None):
     author_email="mark.a.wieczorek@gmail.com",
     license='BSD',
     keywords=KEYWORDS,
-    requires=INSTALL_REQUIRES,
+    install_requires=INSTALL_REQUIRES,
```

Comment:
That doesn't make sense; you need a runtime requirement after it's built, not an install-time requirement.

Reply:
Runtime requirements are deprecated.

Reply:
There are to phases to Python distribution:
1. The build phase, which can specify requirements with PEP 518.
2. The install phase, which can specify requirements with METADATA.

If we build against a certain numpy version, then we require a version of numpy equal or greater to that version to be installed.

Reply:
Yes, I understand why this PR is necessary.

Sorry, I was confusing `install_requires` with `setup_requires` here.